core/src/main/java/tc/oc/pgm/listeners/AntiGriefListener.java

```diff
@@ -90,12 +90,25 @@ private void participantDefuse(Player player, Entity entity) {
                     "moderation.defuse.player",
                     clicker.getBukkit(),
                     owner.getBukkit().getDisplayName(clicker.getBukkit()) + ChatColor.RED));
+
+        ChatDispatcher.broadcastAdminChatMessage(
+            TranslatableComponent.of(
+                "moderation.defuse.alert.player",
+                TextColor.WHITE,
+                clicker.getName(NameStyle.FANCY),
+                owner.getName(NameStyle.FANCY)),
```

Comment:
```suggestion
                owner.getName(NameStyle.FANCY),
                MinecraftTranslations.getEntity(entity.getType()).color(TextColor.DARK_RED)),
```
Add the translatable component for the TNT name